src/reference/asciidoc/gemfire.adoc

```diff
@@ -210,3 +210,5 @@ The _Twitter Inbound Channel Adapter_ and the _Feed Inbound Channel Adapter_ wil
 NOTE: The `GemfireMetadataStore` also implements `ConcurrentMetadataStore`, allowing it to be reliably shared across multiple application instances where only one instance will be allowed to store or modify a key's value.
 These methods give various levels of concurrency guarantees based on the scope and data policy of the region.
 They are implemented in the peer cache and client/server cache but are disallowed in peer Regions having NORMAL or EMPTY data policies.
+
+NOTE: Since _Spring Integration 5.0_, the `GemfireMetadataStore` also implements `ListenableMetadataStore`, allowing users to listen to cache events by providing `MetadataStoreListener` instances to the store.
```

Comment:
Well, we definitely in Spring Integration. To say `_version 5.0_` is enough.
If that is not about Spring Integration we would say that, surely 😄